### ISSUE TITLE
MSCalendarView missing startDateTime & endDateTime

### ISSUE THREAD

**Issue by nawar** [microsoftgraph/msgraph-sdk-ios#17]
I noticed that the `MSGraphCalendarView` is missing `startDateTime` and `endDateTime` when trying to build its request. It's strange that this class is missing those whereas a class of `MSGraphUserReminderViewRequest` has them.

```objective-c

@class MSURLSessionDataTask;

#import "MSCollectionRequest.h"

@interface MSGraphUserReminderViewRequest : MSCollectionRequest

@property (readonly) NSMutableURLRequest *mutableRequest;

- (instancetype)initWithStartDateTime:(NSString *)startDateTime endDateTime:(NSString *)endDateTime URL:(NSURL *)url options:(NSArray *)options client:(ODataBaseClient*)client;

- (MSURLSessionDataTask *)executeWithCompletion:(void (^)(MSCollection *response, MSGraphUserReminderViewRequest *nextRequest, NSError *error))completionHandler;

@end
```

and `MSGraphCalendarCalendarViewCollectionRequestBuilder `

```objective-c
@class MSGraphCalendarCalendarViewCollectionRequest, MSGraphEventRequestBuilder;

#import "MSGraphModels.h"
#import "MSCollectionRequestBuilder.h"




@interface MSGraphCalendarCalendarViewCollectionRequestBuilder : MSCollectionRequestBuilder

- (MSGraphCalendarCalendarViewCollectionRequest *)request;

- (MSGraphCalendarCalendarViewCollectionRequest *)requestWithOptions:(NSArray *)options;

- (MSGraphEventRequestBuilder *)event:(NSString *)event;

@end
```

In the Graph docs, you can see that CalendarView sends the start and end dates

https://graph.microsoft.io/en-us/docs/api-reference/v1.0/api/calendar_list_calendarview

**Comment by dpim** [comment]
I believe this is a case of the generated models being out-of-date with the service. We'll track this as an issue.

**Comment by Cliff2103** [comment]
Fixed this in #34

**Comment by jasonjoh** [comment]
This project is being archived and replaced with the [Microsoft Graph SDK for ObjC](https://github.com/microsoftgraph/msgraph-sdk-objc). As part of the archival process, we're closing all open issues and pull requests. See #37 for more context.

You can continue to use this library "as-is", but it won't be maintained moving forward. We apologize for any inconvenience.
